### DISCUSSION TITLE
Fuzz request parameters

### DISCUSSION THREAD

**Discussion by cfsdes** [projectdiscovery/nuclei#726] [Ideas]
Please add the option of fuzzing on request parameters, such as the jaeles generator option. I know we already have a fuzzing option, but it is very important a feature capable of automatically identifying the request parameters and inserting our wordlist there.

The functionality would basically allow us to set a single value in a variable or a wordlist and this value can be applied to each request parameter that it identifies.

This is essential for injection vulnerabilities

**Comment by ehsandeep** [comment]
@cfsdes this is something already added in the following branch - https://github.com/projectdiscovery/nuclei/tree/fuzzing-support, soon we will be working to include it in the main branch.

**Comment by MetzinAround** [comment]
AFAIK, we now have fuzzing options.
